app/path.ts

```diff
@@ -1,7 +1,7 @@
 export enum Path {
   Index = "/",
   Home = "/home",
-  Send = "/send",
+  Send = "/send/:tokenAddress",
```

Comment:
Perhaps we can concat the variable in the send page? Adding it here would make it unavailable to use path for navigate like `navigate(Path.Send)`

Reply:
Sure! 
We still need to define the url schema in the `<PageRouter />`, or it cannot read `/send/${address}`.

[concate variable](https://github.com/pilagod/waallet-extension/pull/170/commits/09cc97da15fb9a282cce65c7023ac31b3960a782)

Reply:
You're right. I saw so many places need the concat 🥲 I don't have good idea to organize them now, we can refine them when we get a good way.